dataEmbedder.py

```diff
@@ -54,15 +58,24 @@ def read_config(self, config: str = EMBED_CONFIG_PATH) -> str:
 
         :param config: 配置文件路径字符串
         """
-        pass
+        keyList = []
+        with open(config) as f:
+            for read_data in f.readlines():
+                read_data = read_data.strip('\n')
+                key = str(read_data)
+                keyList.append(key)
+        f.closed
+        listLen = len(keyList)
+        key = keyList[random.randint(0,listLen-1)]
+        return key
```

Comment:
为什么要从多个密钥里随机取？无法保证提取信息时用的密钥与嵌入信息时的密钥一致。
另外，文件以字节模式读取。
```suggestion
        with open(config, 'rb') as f:
            key = f.read()
        return key
```